extensions-contrib/opencensus-extensions/src/test/java/org/apache/druid/data/input/opencensus/protobuf/OpenCensusProtobufReaderTest.java

```diff
@@ -327,8 +327,9 @@ public void testDimensionSpecExclusions() throws IOException
   }
 
   @Test
-  public void testInvalidProtobuf() throws IOException {
-    byte[] invalidProtobuf = new byte[] { 0x00, 0x01 };
+  public void testInvalidProtobuf() throws IOException
+  {
+    byte[] invalidProtobuf = new byte[] {0x00, 0x01};
```

Comment:
let's keep unrelated formatting changes out of this PR

Reply:
then let's create a separate PR for the formatting changes, and merge it then since it would affect all PRs, not just this one. This way we can keep this PR clean

Reply:
Created https://github.com/confluentinc/druid/pull/109 for the same